languages/ruby/exercises/concept/conditionals-ternary/.meta/example.rb

```diff
@@ -0,0 +1,22 @@
+class Moviegoer
+  attr_reader :age, :movie_club_member
+
+  def initialize(age, movie_club_member: false)
+    @age = age
+    @movie_club_member = movie_club_member
+  end
+
+  # Do not edit above methods, add your own methods below.
```

Comment:
This can be removed from the example.

Reply:
Not done here, we may do this separately.